Scully build fails immediately in new project

## 🐞 Bug report

### Description

I have built a fresh angular application and followed the scully docs on how to add scully and build a static site. 

However when running `npm run scully` I get the following error

```
> scully

using plugins from folder "./scully"
Typescript error while compiling plugins. the error is:
error TS5042: Option 'project' cannot be mixed with source files on a command line.

```

#### 💻Your Environment
Windows 

#### 💻Minimal reproduction steps
```
ng new my-app
cd ./my-app
ng add @scullyio/init
npm run build
npm run scully

```
**Angular Version:**

<pre><code>

     _                      _                 ____ _     ___ 
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | | 
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | | 
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 10.1.3
Node: 12.18.3
OS: win32 x64

Angular: 10.1.3
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------       
@angular-devkit/architect       0.1001.3
@angular-devkit/build-angular   0.1001.3
@angular-devkit/core            10.1.3
@angular-devkit/schematics      10.1.3
@angular/language-service       10.1.4
@schematics/angular             10.1.3
@schematics/update              0.1001.3
rxjs                            6.6.3
typescript                      4.0.3

</code></pre>

**Scully Version:**

<pre><code>
    "@scullyio/init": "^1.0.1",
    "@scullyio/ng-lib": "^1.0.0",
    "@scullyio/scully": "^1.0.0",

</code></pre>

## 🔥 Exception or Error

<pre><code>
> scully

using plugins from folder "./scully"
Typescript error while compiling plugins. the error is:
error TS5042: Option 'project' cannot be mixed with source files on a command line.
</code></pre>

--- a/docs/faq.md
+++ b/docs/faq.md
@@ -76,8 +76,21 @@ To correct this, add the `skipLibCheck` and `skipDefaultLibCheck` flags to your 
 
 > Running scully gives a fatal error: `Unknown type "myPlugin" in route "/aRoute"`
 
-This might happen if you have installed Scully globally, and you are trying to run with the global version.
-Make sure you run scully from the local repo.
+> I get this error:
+
+```
+--------------------------------------------------------------------------
+you started scully outside of a scully project-folder,
+or didn't install packages in this folder.
+We can't find your local copy to start.
+This can also happen on windows with PowerShell and mixed case path-names
+--------------------------------------------------------------------------
+```
+
+This might happen when you started scully from within a different project, a subfolder that is too deeply nested.
+Or you are on Windows, using Powershell and have a uppercase character in your path.
+Scully will first try to start the local version, but if it can't find that, it errors out with this error.
+The solution is that you should start Scully inside the root of your project with:
 
 ```bash
 npm run scully

--- a/libs/scully/src/scully.ts
+++ b/libs/scully/src/scully.ts
@@ -3,7 +3,7 @@
 /**
  * The above line is needed to be able to run in npx and CI.
  */
-import { readFileSync } from 'fs-extra';
+import { existsSync, readFileSync } from 'fs-extra';
 import open from 'open';
 import { join } from 'path';
 import './lib/pluginManagement/systemPlugins';
@@ -31,14 +31,21 @@ if (process.argv.includes('version')) {
   process.exit(0);
 }
 
-if (process.argv.includes('test')) {
-  console.log('starting from', __dirname);
-  if (!__dirname.includes(process.cwd())) {
-    // const { execSync } = require('child_process');
+if (!__dirname.includes(process.cwd())) {
+  /** started from outside project folder, _or_ powershell with uppercase pathname */
+  if (existsSync('./node_modules/@scullyio/scully/scully.js')) {
     execSync('node ./node_modules/@scullyio/scully/scully.js', {
       cwd: './',
       stdio: 'inherit',
     });
+  } else {
+    logError(`
+--------------------------------------------------------------------------
+you started scully outside of a scully project-folder,
+or didn't install packages in this folder.
+We can't find your local copy to start.
+This can also happen on windows with PowerShell and mixed case path-names
+--------------------------------------------------------------------------`);
   }
   process.exit(0);
 }

